scripts/load_testing/locustfile.py

```diff
@@ -16,14 +18,15 @@
 
 # This should match however many users were created
 # for the DB by the rake task.
-NUM_USERS = 100
+NUM_USERS = 1000
+
 
 def random_cred():
     """
     Given the rake task:
-    rake dev:random_users NUM_USERS=1000 SCRYPT_COST='800$8$1$'
+    rake dev:random_users NUM_USERS=100 SCRYPT_COST='800$8$1$'
 
-    We should have 1000 existing users with credentials matching:
+    We should have 100 existing users with credentials matching:
     * email address testuser1@example.com through testuser1000@example.com
```

Comment:
should this be changed to `throught testuser100@example.com`?

Reply:
Good eye. I set it all to 1000 to match NUM_USERS above.